cmd/login.go

```diff
@@ -89,7 +90,12 @@ to log in to a Okteto Enterprise instance running at okteto.example.com.
 				log.Success("Logged in as %s @ %s", u.GithubID, oktetoURL)
 			}
 
-			log.Hint("    Run `okteto namespace` to switch your context and download your Kubernetes credentials.")
+			err = namespace.RunNamespace(ctx, "")
+			if err != nil {
+				return err
```

Comment:
instead of returning the err could we ask the user to run the command independently? (since the login was a success)